packages/sanity/src/router/types.ts

```diff
@@ -36,24 +61,67 @@ export type RouteChildren =
  * @public
  */
 export interface RouterNode {
+  /**
+   * The route information for this node. See {@link Route}
+   */
   route: Route
+  /**
+   * An optional scope for this node.
+   */
   scope?: string
+  /**
+   * An optional object containing transforms to apply to this node.
+   * See {@link RouteTransform} and {@link RouterState}
+   */
   transform?: {
     [key: string]: RouteTransform<RouterState>
   }
+  /**
+   * The child nodes of this node. See {@link RouteChildren}
+   */
   children: RouteChildren
 }
 
 /**
  * @public
  */
 export interface Router extends RouterNode {
+  /**
+   * Indicates whether this router is a route.
+   * @internal
+   */
   _isRoute: boolean
```

Comment:
Not sure if this is actually `@internal` but I marked it assuming the use of `_isRoute` on tsdoc it gets ignored right now. Let me know if I need to change this